tensorboard/webapp/metrics/store/metrics_types.ts

```diff
@@ -150,6 +159,10 @@ export type CardToPinnedCard = Map<NonPinnedCardId, PinnedCardId>;
 
 export type PinnedCardToCard = Map<PinnedCardId, NonPinnedCardId>;
 
+export type CardToMinMax = Map<NonPinnedCardId, MinMaxStep>;
```

Comment:
Yeah, I guess that makes more sense? All the types of cardIds are intercompatible because they are all just wrappers for string.  They are really only lexically different.
```
export type NonPinnedCardId = string;

export type PinnedCardId = string;

export type CardId = NonPinnedCardId | PinnedCardId;
```